### PR TITLE
shut up, bothering bower!

### PR DESCRIPTION
**Pull request by linki**
bower asks for permission to send usage reports, which disturbs the installation process

```
[?] May bower anonymously report usage statistics to improve the tool over time? No
```

tell him to shut up with this

also use npm's bin command to get the local bin folder https://www.npmjs.org/doc/cli/npm-bin.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "bower": "~1.3.8"
   },
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start --single-run --browsers Firefox,PhantomJS",
-    "postinstall": "./node_modules/bower/bin/bower install"
+    "test": "$(npm bin)/karma start --single-run --browsers Firefox,PhantomJS",
```
Comment (myabc) [inline package.json]: :+1:

Comment (myabc) [inline package.json]: @linki looks like we'll need to provide the `--dev` flag to `npm install` to make this work:

``` diff
diff --git a/.travis.yml b/.travis.yml
index 306426a..b019453 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -56,7 +56,7 @@ before_install:
   - "nvm use 0.10"
   - "travis_retry npm -g install bower@~1.3.8"
   - "travis_retry bower install -F"
-  - "if [[ $TEST_SUITE == 'karma' ]]; then travis_retry npm install; fi"
+  - "if [[ $TEST_SUITE == 'karma' ]]; then travis_retry npm install --dev; fi"
 before_script:
   - "if [[ $TEST_SUITE != 'karma' ]]; then RAILS_ENV=production bundle exec rake ci:travis:prepare; fi"
 notifications:
```

Comment (myabc) [inline package.json]: better yet `npm install --dev --no-optional`

I don't know why npm can't have sensible defaults.

Comment (linki) [inline package.json]: thanks for the hint @myabc

**Comment by mfrister** [comment]
Travis doesn't like this, even after a restart.

**Comment by ulferts** [comment]
@linki any update on this?

**Comment by linki** [comment]
@ulferts I'll check again later. looks like I don't even get the message in my containers, anymore. don't know what's going on, we might don't need this PR at all.

**Comment by tessi** [comment]
I got the message on a recent release/4.0 branch recently. And it bothers me too (it's one step more in the installation guide after all ;))

**Comment by ulferts** [comment]
:+1:

**Comment by linki** [comment]
thx, squashing commits would have been great :palm_tree:

**Pull request merged**